pkg/collector/check/jmx.go

```diff
@@ -13,16 +13,18 @@ import (
 )
 
 // IsJMXConfig checks if a certain YAML config is a JMX config
-func IsJMXConfig(name string, initConf integration.Data) bool {
+func IsJMXConfig(name string, initConf integration.Data, rawInitConfig integration.RawMap) bool {
```

Comment:
I don't understand why this change is still necessary 🤔

Reply:
Removed...